sqlmesh/core/engine_adapter/base.py

```diff
@@ -263,17 +263,14 @@ def _create_table_from_df(
         replace: bool = False,
         **kwargs: t.Any,
     ) -> None:
-        table = exp.to_table(table_name)
         if not isinstance(df, pd.DataFrame):
             raise ValueError("df must be a pandas DataFrame")
-        expression = next(
```

Comment:
This logic was wrong. If he number of rows exceeded the `DEFAULT_BATCH_SIZE` of the adapter we'd write partial data.